nbconvert/templates/latex/document_contents.tplx

```diff
@@ -35,7 +35,7 @@
 
 % Display markdown
 ((* block data_markdown -*))
-    ((( output.data['text/markdown'] | citation2latex | strip_files_prefix | convert_pandoc('markdown+tex_math_double_backslash', 'latex'))))
+    ((( output.data['text/markdown'] | citation2latex | strip_files_prefix | convert_pandoc('markdown+tex_math_double_backslash', 'latex', extra_args=[], relative_path_replacement=resources.working_directory, build_path_replacement=resources.output_files_dir))))
```

Comment:
These values are set in `_fake_output_files_dir` in `exporters/pdf.py`. This was the easiest way for the wrapping process to inform the conversion that we're operating in a different directory than the file, and to provide a latex-safe path for conversions to take place within.